April 2024 release blog post (#34)
ecamp/ecamp-site: src/content/posts/2024-04-release.en.md

---
title: April release
path: 2024-04-release
pubDate: 2024-04-14
description: Accept invitations directly in the app, copy categories, and more
image: '~/assets/images/080720_lotos_7896_small.jpg'
---

Barely three weeks have passed since the last update of eCamp v3, and already we bring you the next one. This time, the following features are included:

<div class="simple-columns bg-slate-100 dark:bg-slate-800">

![Accept invitations in the UI](~/assets/images/personal-invitations-en.png)

<div>

> Invitations to camps can now also be accepted directly on the eCamp v3 application. For this to work, your camp leaders need to invite the exact email address belonging to your user account. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink}

Reviewer: ```suggestion
Invitations to camps can now also be accepted directly in eCamp v3, given the invitation was sent to the email address of your user account. [#4865](https://github.com/ecamp/ecamp3/issues/4865){.issuelink}
```
